schoolopy/main.py

```diff
@@ -11,18 +11,18 @@
 
 
 class Schoology:
-    _ROOT = 'https://api.schoology.com/v1/'
     key = ''
     secret = ''
     limit = 20
     start = 0
 
-    def __init__(self, schoology_auth):
+    def __init__(self, schoology_auth, api_server='https://api.schoology.com/v1'):
```

Comment:
I would prefer the variable be called `host` or `api_host`

Reply:
Fixed in https://github.com/gadhagod/schoolopy/commit/46940ca43f68dff35f7de047dea6735a0ae55309.